go-apps/meep-loc-serv/sbi/loc-serv-sbi.go

```diff
@@ -159,17 +161,17 @@ func processActiveScenarioUpdate() {
 		uePerNetLocMap[netLoc]++
 	}
 
-	// Update POA info
-	poaNameList := sbi.activeModel.GetNodeNames("POA")
+	// Update POA-CELL info
+	poaNameList := sbi.activeModel.GetNodeNames("POA-CELL")
```

Comment:
RNIS requires Cellular POAs. Is the same true for the Location Service? If not then we should support both generic & cellular POAs here.

Reply:
Does this change break cypress tests?

Reply:
no change to cypress... this is only on which POA type location service check before increasing increasing the number of POAs in status report... I on purpose only put the cellular ones... This wasnot updated after cellular type was added. Location service is only meant for cellular POAs